packages/bot-web-ui/src/stores/google-drive-store.ts

```diff
@@ -121,6 +138,21 @@ export default class GoogleDriveStore implements IGoogleDriveStore {
         this.is_authorised = is_signed_in;
     }
 
+    async verifyGoogleDriveAccessToken() {
+        const expiry_time = localStorage?.getItem('google_access_token_expiry');
+        if (expiry_time) {
+            const current_epoch_time = Math.floor(Date.now() / 1000);
+            if (current_epoch_time > Number(expiry_time)) {
+                this.signOut();
+                this.setGoogleDriveTokenValid(false);
```

Comment:
are we not trying to login the user again automatically, when his token has expired. Or every 24 hours we will be asking him to login right ?

Reply:
done ✅